app/presenters/organisations/header_presenter.rb

```diff
@@ -0,0 +1,118 @@
+module Organisations
+  class HeaderPresenter
+    attr_reader :org
+
+    def initialize(organisation)
+      @org = organisation
+    end
+
+    def organisation_logo
+      component_data = {}
+      component_data[:name] = org.formatted_title.html_safe if org.formatted_title
+      component_data[:brand] = org.brand if org.brand
+      component_data[:crest] = org.crest if org.crest
+
+      if org.organisation_image_url
+        component_data[:image] = {
+          url: org.organisation_image_url,
+          alt_text: org.organisation_image_alt_text
+        }
+      end
+
+      component_data
+    end
+
+    def ordered_featured_links
+      links = []
+
+      if has_services_and_information_link?
+        see_more_link = {
+          text: "All #{org.title} services and information", # need to use acronym where available
```

Comment:
It's not clear this is something that still needs to be implemented. Can we add ` TODO: ` to make it clearer/reference a Trello card?